src/views/ev3/ev3.jsx

```diff
@@ -279,8 +280,10 @@ class EV3 extends ExtensionLanding {
                         />
                     </Steps>
                 </ExtensionSection>
-                <ExtensionSection className="faq">
-                    <h2><FormattedMessage id="ev3.troubleshootingTitle" /></h2>
+                <ExtensionTroubleshooting
+                    deviceName={this.props.intl.formatMessage({id: 'ev3.deviceName'})}
```

Comment:
I don't think this actually needs to be in a `formatMessage`. Most of the device names are brand/trademarks that we don't want people to translate.